test/strategies/base.ts

```diff
@@ -69,6 +69,74 @@ describe('Strategy', () => {
       expect(pullRequest?.version?.toString()).to.eql('2.3.4');
       snapshot(dateSafe(pullRequest!.body.toString()));
     });
+    it('updates extra files', async () => {
+      const strategy = new TestStrategy({
+        targetBranch: 'main',
+        github,
+        component: 'google-cloud-automl',
+        extraFiles: [
+          '0',
+          'foo/1.~csv',
+          'foo/2.bak',
+          'foo/baz/bar/',
+          '/3.java',
+          '~/4.md',
+          '~/./5',
+        ],
+      });
+      const pullRequest = await strategy.buildReleasePullRequest(
+        [{sha: 'aaa', message: 'fix: a bugfix'}],
+        undefined
+      );
+      expect(pullRequest).to.exist;
+      expect(pullRequest?.updates).to.be.an('array');
+      expect(pullRequest?.updates.map(update => update.path))
+        .to.include.members([
+          '0',
+          '3.java',
+          '4.md',
+          './5',
```

Comment:
AFAIK, this will end up making an invalid request to the GitHub API (fetching a file at path `./5`). Someone configuring this path should just use `5`.

Reply:
It will now throw an error in f9968f8025443b3aa062794e6a37b21d4e2e56b9